src/leveldb/benchmarks/db_bench.cc

```
@@ -283,33 +278,31 @@ class Stats {
   //    (4) done
 
-  int num_initialized;
-  int num_done;
-  bool start;
+  int num_initialized GUARDED_BY(mu);
+  int num_done GUARDED_BY(mu);
+  bool start GUARDED_BY(mu);
 
-  SharedState() : cv(&mu) { }
+  SharedState(int total)
+      : cv(&mu), total(total), num_initialized(0), num_done(0), start(false) {}
```

cppcheck (2.10-2)

[error] Member variable 'total' is initialized by itself.

(selfInitialization)